cmd/controller-manager/app/controllermanager.go

```diff
@@ -592,9 +592,6 @@ func startGracefulEvictionController(ctx controllerscontext.Context) (enabled bo
 }
 
 func startApplicationFailoverController(ctx controllerscontext.Context) (enabled bool, err error) {
-	if !features.FeatureGate.Enabled(features.Failover) {
-		return false, nil
-	}
```

Comment:
It makes sense to me that this patch won't affect users who are using application failover.

Reply:
yes, it makes sense.
/lgtm